### DISCUSSION TITLE
Cotton Farming < Cotton Ginning

### DISCUSSION THREAD

**Discussion by Wab1l** [USEPA/supply-chain-factors#85] [Q&A] [answered]
The Cotton Farming 111920 has a larger emission factor than Cotton Ginning 115111, how is this possible as cotton farming is part of of the supply chain of Cotton ginning so it should have a large emission factor the Cotton Farming? Explain Please

**Comment by bl-young** [comment, marked as answer]
All factors are provided per dollar and reflect the upstream supply chain (i.e., cradle-to-gate, or cradle-to-retail). There is no specific requirement that the factors (per dollar) increase along each stage of the supply chain. Consider that purchased goods and raw materials are just one component of a sector's supply chain, and the value added by an industry impacts the overall economic output of that sector. As an example, imagine Industry B only consumes outputs of Industry A, but processes them into higher value products (with no other emissions or energy inputs). The emission factor for Industry B would likely be _lower_ than Industry A because it has the same total emissions relative to a higher economic output.

In this particular case, I'll add that Cotton Ginning is within the "115000 Support activities for agriculture and forestry" sector, so I don't think the supply chain manifests in the way you describe (that is, 115000 does not make substantial purchases from "111900 Other crop farming")

**Reply by Wab1l** [reply]
Thanks for the guidance @bl-young
